common/sketch/src/main/java/org/apache/spark/util/sketch/BloomFilterImpl.java

```diff
@@ -221,6 +221,49 @@ public BloomFilter mergeInPlace(BloomFilter other) throws IncompatibleMergeExcep
   }
 
   @Override
+  public double approxItems() {
+    double m = bitSize();
+    return (-m / numHashFunctions) * Math.log(1 - (bits.cardinality() / m));
```

Comment:
* `Math` is deprecated. Use `math`.
* Please add a test when `bits` is full. This should return `Double.Infinity`.